test/bdd/fixtures/wallet-web/docker-compose.yml

```diff
@@ -200,8 +200,8 @@ services:
       - HTTP_SERVER_RP_DISPLAY_NAME=trustbloc
       - HTTP_SERVER_RP_ORIGIN_NAME=https://bddtest-wallet-web.trustbloc.local:8077/
       - HTTP_SERVER_RP_ID=localhost
-      - HTTP_SERVER_AUTHZ_KMS_URL=https://authz-kms.trustbloc.local:${AUTHZ_HUB_KMS_PORT}
-      - HTTP_SERVER_OPS_KMS_URL=https://ops-kms.trustbloc.local:${OPS_HUB_KMS_PORT}
+      - HTTP_SERVER_AUTHZ_KMS_URL=https://bdd-oathkeeper-auth-keyserver.trustbloc.local:4459
+      - HTTP_SERVER_OPS_KMS_URL=https://bdd-oathkeeper-ops-keyserver.trustbloc.local:4460
```

Comment:
We need to update [demo wallets](https://github.com/trustbloc/edge-agent/blob/master/test/bdd/fixtures/wallet-web/docker-compose.yml#L121-L122) to use oathkeeper as well, right ? Is that planned for future PR ?

Reply:
It was broken yesterday - see #560